pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```
@@ -0,0 +1,87 @@
+{ lib, stdenv, fetchFromGitHub, buildLinux, ... } @ args:
+
+let
+  stableVariant =
+    let release = "1";
+    in
+    {
```

```suggestion
  stableVariant = {
```

Don't need this indirection.